mail_tracking/models/mail_thread.py

```diff
@@ -0,0 +1,44 @@
+# Copyright 2019 Alexandre Díaz
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import models, api
+from email.utils import COMMASPACE
+
+
+class MailThread(models.AbstractModel):
+    _inherit = "mail.thread"
+
+    @api.multi
+    @api.returns('self', lambda value: value.id)
+    def message_post(self, body='', subject=None, message_type='notification',
+                     subtype=None, parent_id=False, attachments=None,
+                     content_subtype='html', **kwargs):
+        new_message = super().message_post(
+            body=body, subject=subject, message_type=message_type,
+            subtype=subtype, parent_id=parent_id, attachments=attachments,
+            content_subtype=content_subtype, **kwargs)
+        email_cc = kwargs.get('cc')
+        if email_cc:
+            new_message.sudo().write({
+                'email_cc': email_cc,
+            })
+        return new_message
+
+    @api.multi
+    def message_get_suggested_recipients(self):
+        res = super().message_get_suggested_recipients()
+        ResPartnerObj = self.env['res.partner']
+        for record in self:
+            messages = record.message_ids.filtered(lambda x: x.email_cc)
+            for msg in messages:
+                email_cc = msg.email_cc.split(COMMASPACE)
+                for cc in email_cc:
+                    partner_id = record.message_partner_info_from_emails(
+                        [cc])[0].get('partner_id')
+                    if not partner_id:
+                        res[record.ids[0]].append((False, cc, "Cc"))
+                    else:
+                        partner = ResPartnerObj.browse(partner_id)
+                        record._message_add_suggested_recipient(
+                            res, partner=partner, reason='Cc')
```

Comment:
I'm also not sure about this, but possibly it's needed. Could you check it please?:

```suggestion
                            res, partner=partner, reason=_('Cc'))
```